libs/gi/formula/src/index.spec.ts

```diff
@@ -107,16 +112,15 @@ describe('sheet', () => {
             if (sheet === 'iso' || sheet === 'agg' || sheet === tag.sheet)
               continue
             fail(`Illform entry (${tagStr(tag)}) for sheet ${sheet}`)
-            break
           }
+          // eslint-disable-next-line no-fallthrough
           case 'enemy': {
             const desc = (enemyTag as any)[tag.qt]?.[tag.q]
             if (!desc) continue
             const { sheet } = desc
             if (!sheet) continue
             if (sheet === 'agg' || sheet === tag.sheet) continue
             fail(`Illform entry (${tagStr(tag)}) for sheet ${sheet}`)
-            break
```

Comment:
Would removing break here cause more issues since logic would go to the next case?

Reply:
`fail` throws an error, so the break is unreachable and thus removed by linter. but then linter complains about fallthrough which does not exist